src/main/java/persistence/entity/EntityPersister.java

```diff
@@ -0,0 +1,32 @@
+package persistence.entity;
+
+import jdbc.JdbcTemplate;
+import persistence.sql.ddl.PrimaryKeyClause;
+import persistence.sql.dml.DeleteQueryBuilder;
+import persistence.sql.dml.InsertQueryBuilder;
+import persistence.sql.dml.UpdateQueryBuilder;
+
+public class EntityPersister {
+    private final JdbcTemplate jdbcTemplate;
+
+    public EntityPersister(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public boolean update(Object entity, Long id) {
+        int result = jdbcTemplate.executeUpdate(new UpdateQueryBuilder(entity.getClass()).getQuery(entity, id));
+        return result == 1;
+    }
```

Comment:
사소하지만 다른 메서드들 처럼 query 부분이 변수로 빠지는게 가독성이 더 좋은것 같아요 👍 
```suggestion
    public boolean update(Object entity, Long id) {
        String query = new UpdateQueryBuilder(entity.getClass()).getQuery(entity, id);
        int result = jdbcTemplate.executeUpdate(query);
        return result == 1;
    }
```